driver-adapters/planetscale-lambda-basic/index.js

```diff
@@ -40,6 +40,30 @@ exports.handler = async () => {
           },
         ],
       }),
+      // Not implemented for MySQL
```

Comment:
I think we can remove this completely for now: as far as we concerned, with current state of things this can not be implemented for mysql at all and very likely we'll never need this test ate all.